ERC-7913 restructuring (#5)
Amxx/ERCs: ERCS/erc-7913.md

### Nomenclature

> - Keys are described as a `bytes` object of arbitrary length. For example a P256 or RSA public key.

Reviewer: ```suggestion
- Keys are described using `bytes` objects of arbitrary length. For example a P256 or RSA public key. Keys MUST NOT be empty.
```

}
```

> Verifiers SHOULD NOT be stateful.

Reviewer 2: Isn't it clearer?

```suggestion
Verifiers SHOULD be stateless.
```

## Backwards Compatibility

> A system can support ERC-7913 signers alongside EOAs and ERC-1271 in the following way.

Reviewer: ```suggestion
A system can support [ERC-7913](./eip-7913.md) signers alongside EOAs and ERC-1271 in the following way.
```

> A system can support ERC-7913 signatures alongside EOAs and ERC-1271 in the following way.

Reviewer: ```suggestion
A system can support [ERC-7913](./eip-7913.md) signatures alongside EOAs and ERC-1271 in the following way.
```

Author: Accepted without the link since a link to itself seems odd.

Reviewer: If we write ERC-7913, then we need the link !

There is a "rule" for eip document enforces that the first reference to an ERC/EIP must be the link. That rule also apply to self. If you don't want the link you need to rephrase things.
